nominations/iulian-dragos.md

```diff
@@ -0,0 +1,22 @@
+# Iulian Dragos @dragos
+
+## Proposer
+
+This candidate has been nominated by @fommil and seconded by @rorygraves.
+
+## Qualifications and Experience
+
+- expert on the scala interactive compiler
+- wrote the first scala LSP implementation
+
+## Affiliations
+
+- standing as an individual
+
+## Working Groups
+
+This candidate is nominated as a candidate to the LSP Working Group (LSP-WG).
```

Comment:
@dragos may modify which of the two groups he feels best representing. If in doubt, include both.